superset/charts/schemas.py

```diff
@@ -304,6 +304,21 @@ class ChartCacheScreenshotResponseSchema(Schema):
     image_url = fields.String(
         metadata={"description": "The url to fetch the screenshot"}
     )
+    update_status = fields.String(
+        metadata={"description": "The status of the async screenshot"}
+    )
+    updated_at = fields.String(
+        metadata={"description": "The timestamp of the last change in status"}
+    )
```

Comment:
"update" seems ambiguous here:
```suggestion
    task_status = fields.String(
        metadata={"description": "The status of the async screenshot task"}
    )
    task_updated_at = fields.String(
        metadata={"description": "The timestamp of the last change in task status"}
    )
```